docs/user/guides/setup/index.rst

```diff
@@ -0,0 +1,45 @@
+How-to guides: project setup and configuration
+==============================================
+
+The following how-to guides help you solve common tasks and challenges in the setup and configuration stages.
+
+⏩️ :doc:`Connect your Read the Docs account to your Git provider </guides/connecting-git-account>`
+    Steps to connect an account on |git_providers_or| with your Read the Docs account
+
+⏩️ :doc:`Connect your Git repository automatically </connected-accounts>`
+    Connecting your Read the Docs account to one of the supported Git providers, |git_providers_or|.
+
+⏩️ :doc:`Connect your Git repository manually </guides/git-integrations>`
+    Setting up and connecting your Git repository manually, supports most Git providers with webhooks.
```

Comment:
It's really not clear to me how these 3 pages differ, as a reader?

Reply:
It's the same as before. At least now, there's a sentence accompanying each guide :) I think we should solve it separately in this case.

I don't think we can improve much without restructuring the guides? I could imagine that merging the first 2 guides would be beneficial, since "Connect your Read the Docs account to your Git provider" is a precondition for "Connect your Git repository automatically".

Having 2 guides: "Doing things automatically" vs. "Doing things manually" will be less work for the reader to digest.

Reply:
I just think users want to do the thing, right? The manual & automatic approaches should presumably be in the same place? I just feel like we broke up our guides in a way that makes them much harder for users to use, and we need to undo that if we're finding it confusing, since users will be even more confused.

Reply:
I think that the manual and automatic approaches are different howtos because they solve different problems. I'm not totally happy with it, but I think it improved now by adding much clearer scope on what the manual guide is for:

![image](https://user-images.githubusercontent.com/374612/225664974-eeb97952-406f-4935-a08e-2951e1d47b56.png)